gym/wrappers/normalize.py

```diff
@@ -81,19 +81,20 @@ def step(self, action):
 
     def reset(self, **kwargs):
         """Resets the environment and normalizes the observation."""
-        return_info = kwargs.get("return_info", False)
-        if return_info:
+        if kwargs.get("return_info", False):
             obs, info = self.env.reset(**kwargs)
+
+            if self.is_vector_env:
```

Comment:
This also looks like a change of behavior, is this intended? Can you explain it?

Reply:
I believe that it is not a change in behaviour. The problem previous was that `info` could have been unbounded. To avoid the issue, I have all of the return info code in one part of the if statement and without info is the second part. This has the cost of copying the if vector code twice but it shouldn't be a issue